### DISCUSSION TITLE
Paste image from clipboard into WYSIWYG

### DISCUSSION THREAD

**Discussion by Christilut** [directus/directus#13164] [Feature Requests]
It would be great if I could paste an image directly from the clipboard inside the WYSIWYG/markdown editor.

This would be tremendously helpful when creating guides/tutorials and such. Sometimes you are writing something with a lot of screenshots and it's super cumbersome to:

1. Save it as image on disk
2. Click the image button
3. Drag or find the earlier image
4. Delete the image from disk

Opposed to only pressing CTRL/CMD+V.

I requested this same feature years ago at Prismic but they refused and every time I wrote an article I had this same issue. It's just such a great little thing to be able to do this.

**Comment by Rar9** [comment]
this should be extended for Images/files as well 
Quick example https://jsfiddle.net/ourcodeworld/hzvfq82b/

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Comment by arladmin** [comment]
Has anyone been able to figure out a workaround/hack to implement this feature?

Have a project where users need to screenshot content from pdfs and paste the images in a wysiwyg field. 
And they need to do this for million+ items! 

So, now, they have to resort to first naming & saving every screenshot on their system, and then upload to the wysiwyg editor 🫤

**Comment by marcmojave** [comment]
The copying of images into WYSIWYG was possible in `v9.00-rc.30` but when I try with the same content in `9.26.0`, they dont render. Also, there was a drop down to allow you create anchor tags but that doesn't seem to work on 9.26.0. Is there a config I am missing or was that functionality removed?



https://github.com/directus/directus/assets/25531999/d5a73e4c-3bd3-449b-80f3-8bd925af9def

**Reply by Christilut** [reply]
I don't think you are pasting an image, but HTML which includes the <img tag to the image

**Comment by Christilut** [comment]
This is still the most missed feature for me. Every time I write a blog, I have to do this:
1. Snip the screen, it saves to Documents folder automatically
2. Click the image icon in the wysiwig header (usually scroll up first because the header isn't sticky)
3. Click the Open button in the popup
4. Go to Documents folder
5. Click the image
6. Confirm the image
7. Optional: Delete the image from Documents

Or if I could paste an image, it would be this:
1. Snip the screen, don't save it anywhere
2. Paste in wysiwig

DONE! So insanely much faster and more pleasant to use.
